RemixedDungeon/src/main/java/com/watabou/pixeldungeon/items/Gold.java

```diff
@@ -66,6 +67,7 @@ public boolean doPickUp(@NotNull Char hero ) {
 		hero.spendAndNext( TIME_TO_PICK_UP );
 		
 		Sample.INSTANCE.play( Assets.SND_GOLD, 1, 1, Random.Float( 0.9f, 1.1f ) );
+		new Hook().Call("onGoldPickup", this, hero);
```

Comment:
This is really needed when onItemPickup already here?

Reply:
You are right. I will remove it.